src/applications/disability-benefits/all-claims/pages/medicalCare.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+function noCareCheck(errors, state, formData) {
+  const medicalCare = formData['view:medicalCare'];
+  if (
+    medicalCare.underDoctorCare &&
```

Comment:
Maybe we can use the `_.get()` on these as well and store them in `const` so we can reuse them in the conditionals.

example: `const isHospitalized = _.get('view:medicalCare.hospitalized', data, false);`